source/layouts/layout.html.erb

```diff
@@ -11,7 +11,7 @@
       <!--
         Version: <%= version['version'] %>
         Latest: <%= package.latest %>
-        Is blocklisted?: <%= data.blocklist.include?(package.name) %>
+        Is blocklisted?: <%= data.blocklist.include?(package.namespace + "/" + package.name) %>
```

Comment:
This is the only part that's not working. On the current site, blocklisted packages render like so ([example](https://hub.getdbt.com/fishtown-analytics/dbt-utils/latest/):

<img width="1298" alt="Screen Shot 2020-07-31 at 1 26 45 PM" src="https://user-images.githubusercontent.com/20294432/89060733-823b3d00-d331-11ea-92c0-92bf71d8a932.png">


In the deploy preview, no warning is rendered ([example](https://deploy-preview-249--flamboyant-mcclintock-92ba2d.netlify.app/fishtown-analytics/dbt-utils/latest/)):

<img width="1387" alt="Screen Shot 2020-07-31 at 1 28 22 PM" src="https://user-images.githubusercontent.com/20294432/89060861-bc0c4380-d331-11ea-8904-bd5c1eaba27b.png">

Reply:
this banner is applied when the [is_hidden](https://github.com/fishtown-analytics/hub.getdbt.com/blob/1c9029d6f60ff7b3838bd13b8a4f1fcf74375fe7/source/package.template.html.erb#L12-L18) helper returns true.

The source code for the `is_hidden` helper is:
https://github.com/fishtown-analytics/hub.getdbt.com/blob/1c9029d6f60ff7b3838bd13b8a4f1fcf74375fe7/config.rb#L125-L127